app/src/org/odk/collect/android/views/media/AudioButton.java

```diff
@@ -27,32 +27,26 @@
     private final static String t = "AudioButton";
     private String URI;
     private MediaState currentState;
-    private final AudioController controller;
     private Object residingViewId;
 
+    /**
+     * Used by inflater.
+     */
     public AudioButton(Context context, AttributeSet attrs) {
         super(context, attrs);
```

Comment:
What sort of logic needs to be in constructors that are used by the media inflater?